hapi/hedera-protobufs/block/stream/output/state_changes.proto

```diff
@@ -857,6 +863,11 @@ message MapChangeValue {
          * A change to the scheduled throttle usage snapshots.
          */
         proto.ThrottleUsageSnapshots throttle_usage_snapshots_value = 19;
+
+        /**
+         * A TSS Encryption Key Transaction Body value.
+         */
+        com.hedera.hapi.services.auxiliary.tss.TssEncryptionKeyTransactionBody tss_encryption_key_value = 20;
```

Comment:
This seems very odd and concerning. Are we truly storing full transaction bodies _in state_?  If we are, I would _recommend_ changing that, particularly as this message only has one field (of `bytes`).
We _could_ more clearly store the `bytes` as a value in state with the same "key" used (is it Node ID?) to avoid the added wrapper message and also reduce potential confusion.

Reply:
I think the discussion was resolved in the Protobufs for Tss Startup PR in favor of keeping the transaction bodies in state. Shall we mark this as resolved? @jsync-swirlds